lib/actions/file.js

```diff
@@ -41,7 +44,10 @@ exports.listFileNames = function(b2, args) {
         body: JSON.stringify({
             bucketId: bucketId,
             startFileName: startFileName ? startFileName : '',
-            maxFileCount: maxFileCount ? maxFileCount : 100
+            maxFileCount: maxFileCount ? maxFileCount : 100,
+            prefix: prefix ? prefix : '',
+            delimiter: delimiter ? delimiter : null
```

Comment:
Cheer @crazyscience I'll hold off publishing for the next PR 👍 

That PR and this one will go out as `0.9.12`